.bazelrc

```diff
@@ -10,6 +10,11 @@ build --cxxopt='-Werror' --cxxopt='-Wall'
 build --features=external_include_paths
 build --cxxopt='-Wno-deprecated-declarations'
 
+# These are needed on systems with GLIBC > 2.34-8 to build google-fuzztest
```

Comment:
You might want to move it below where there is already some exceptions related to fuzztest?

You will also need to add an equivalent `--host_cxxopt`

Reply:
fixed